src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/StatusAlert.jsx

```diff
@@ -37,7 +37,7 @@ export default function StatusAlert({ appointment, facility }) {
   } else if (showConfirmMsg) {
     return (
       <InfoAlert backgroundOnly status="success">
-        <strong>Your appointment has been scheduled and is confirmed.</strong>
+        <strong>We've scheduled and confirmed your appointment.</strong>
```

Comment:
Very subtle difference here but I think we are supposed to use the appostrophe character '’' as opposed to a single quote character '''. You can get the appostrophe character on a mac by using shift + option + ].